_data/banking.yml

```diff
@@ -410,6 +410,15 @@ websites:
       img: fifththird.png
       tfa: No
 
+    - name: First Community Credit Union
+      url: https://www.firstcommunity.com/
+      img: first_community_cu.png
+      tfa: Yes
+      sms: Yes
+      phone: Yes
+      email: Yes
```

Comment:
In your `doc` link it only states
> The login security uses a verification code to authenticate yourself via text message and/or a voice call to ensure that it's you trying to access your Online Banking accounts.

No mention of email..Can you confirm this?